### PR TITLE
Mattermost Handbook > Localization Updates

### PR DESCRIPTION
**Pull request by cwarnermm**
Updated the Mattermost Handbook Localization page as follows:
- Added clarification for Beta language support: https://handbook.mattermost.com/contributors/join-us/localization#beta
- Updated Translation Maintenance table: https://handbook.mattermost.com/contributors/join-us/localization#translation-maintenance
   - Correct current Romanian translator reviewer as reviewer for Bulgarian product strings
   - List Romanian language as open roles:

### CODE REVIEW

File: contributors/join-us/localization.md [hunk 1]
```diff
@@ -108,6 +108,7 @@ If you're interested in contributing to the process, please join the [Mattermost
 
 | Language | Official Reviewer\(s\) | Maintainers |
 | :--- | :--- | :--- |
+| Bulgarian | [Nikolai Zahariev \(nikolaiz\)](https://github.com/mattermost/mattermost-handbook/tree/55178956e5d33dde426723e7eb3897c9fcacc52e/contributors/contributors/translate.mattermost.com/user/nikolaiz/README.md) | |
```
Comment (jasonblais) [inline contributors/join-us/localization.md]: Is this intended to link to https://translate.mattermost.com/user/nikolaiz/?

**Review by jasonblais — CHANGES_REQUESTED** [review]

File: contributors/join-us/localization.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ To translate a language:
    * If it is, perhaps offer to help translate or review and make suggestions.
    * If not, ask to setup your language on the [Mattermost localization channel](https://community.mattermost.com/core/channels/localization), so you can start helping with the translation.
 
-   **Note:** Languages are grouped either as "shipped" or "work in progress". A language [ships in Mattermost](https://docs.mattermost.com/help/settings/account-settings.html#language) if it meets specified translation quality.
+   **Note:** Languages are grouped either as "shipped", "work in progress". A language [ships in Mattermost](https://docs.mattermost.com/help/settings/account-settings.html#language) if it meets specified translation quality.
```
Comment (cwarnermm) [inline contributors/join-us/localization.md]:
```suggestion
   **Note:** Languages are grouped either as "shipped" or "work in progress". A language [ships in Mattermost](https://docs.mattermost.com/help/settings/account-settings.html#language) if it meets specified translation quality.
```

File: contributors/join-us/localization.md (hunk 1 above)
Comment (cwarnermm) [reply contributors/join-us/localization.md]: Yes. I'll get that corrected.

Comment (cwarnermm) [inline contributors/join-us/localization.md]:
```suggestion
| Bulgarian | [Nikolai Zahariev \(nikolaiz\)](https://translate.mattermost.com/user/nikolaiz/) | |
```

**Review by jasonblais — APPROVED** [review]

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**